pom.xml

```diff
@@ -82,6 +91,12 @@
       <version>${activiti-cloud-service-common.version}</version>
       <type>pom</type>
     </dependency>
+    <dependency>
+      <groupId>org.activiti.cloud.acc</groupId>
```

Comment:
@igdianov can please check this changes.. I don't remember exactly why we need the dependency here.. I assume that is for the dependency version checks plugin.. but can you please double check

Reply:
@salaboy I don' t think it is needed for dependency convergence validation.